modules/light-clients/07-tendermint/types/upgrade.go

```diff
@@ -14,11 +14,12 @@ import (
 )
 
 // VerifyUpgradeAndUpdateState checks if the upgraded client has been committed by the current client
-// It will zero out all client-specific fields (e.g. TrustingPeriod and verify all data
+// It will zero out all client-specific fields (e.g. TrustingPeriod) and verify all data
 // in client state that must be the same across all valid Tendermint clients for the new chain.
+// Zeroing out the submitted client prevents the proposal from containing information governance is not actually voting on.
```

Comment:
```suggestion
```

My suggestion would be to remove this line. The line above conveys the same idea and I think the real reason for this improvement is to avoid returning an error in the case that the relayer forgets to zero out the client state before submitting it in the message